eng/build.sh

```diff
@@ -31,6 +31,7 @@ usage()
   echo "  --os                            Target operating system: windows, Linux, FreeBSD, OSX, MacCatalyst, tvOS,"
   echo "                                  tvOSSimulator, iOS, iOSSimulator, Android, Browser, NetBSD, illumos or Solaris."
   echo "                                  [Default: Your machine's OS.]"
+  echo "  --packagerid <rid>              Optional argument that overrides the target rid name."
```

Comment:
Maybe it is better to call it `--distrorid` and clear the intent in the description that it is overriding the autodetection of target.

`PackageRID` is meant to restore packages during the build and `OutputRid` is meant to produce shipping packages.

Reply:
> PackageRID is meant to restore packages during the build and OutputRid is meant to produce shipping packages.

This difference is not clear to me. Can you give an example of when `PackageRID` and `OutputRid` are not the same?

Reply:
We can see the difference with `-cross`:

```sh
$ URN="mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-cross-freebsd-12-20220830132358-f13d79e"

$ cd runtime/
$ docker run --rm -e ROOTFS_DIR=/crossrootfs/x64 -v$(pwd):/runtime "$URN" \
  /runtime/build.sh -cross -os freebsd
```

In this case `PackageRID = linux-x64` and `OutputRid = freebsd-x64`.

Reply:
ok. The difference is clear now.

I should be using `OutputRid` instead of `PackageRID` in this PR.